app/src/main/java/org/jellyfin/mobile/fragment/WebViewFragment.kt

```diff
@@ -185,7 +182,27 @@ class WebViewFragment : Fragment(), NativePlayerHost {
                 if (request.url.toString() == instanceUrl) onErrorReceived()
             }
         }
-        webChromeClient = WebChromeClient()
+        webChromeClient = object : WebChromeClient() {
+            override fun onConsoleMessage(consoleMessage: ConsoleMessage?): Boolean {
+                if (consoleMessage?.messageLevel() == null) {
```

Comment:
You should fallback to info if the level is null

Reply:
Actually, you can probably just remove this if check. The message probably never is null.

Reply:
I'm quite sure the reason it's nullable is because the source code is Java and Java doesn't have nullability, so Kotlin makes it nullable by default. I'm quite sure you can just replace the function parameter to use a non-nullable type.

The Android documentation doesn't tell me the value can be null too, while for other methods it does specify that it could be null: https://developer.android.com/reference/android/webkit/WebChromeClient#onConsoleMessage(android.webkit.ConsoleMessage)